api/proto/teleport/legacy/types/events/events.proto

```diff
@@ -3561,10 +3570,10 @@ message LockCreate {
     (gogoproto.jsontag) = ""
   ];
 
-  // Target describes the set of interactions that the lock applies to
-  types.LockTarget Target = 4 [
+  // Lock is a common lock event metadata
+  LockMetadata Lock = 4 [
     (gogoproto.nullable) = false,
-    (gogoproto.jsontag) = "target"
+    (gogoproto.jsontag) = "lock"
```

Comment:
Hmm will renaming this field break protobuf marshal/unmarshal? We should add a new field with metadata to be safe.